tests-web/assets/js/main.js

```diff
@@ -1,46 +1,76 @@
 import { makeDropHandler, makeLinkHandler, handleDragOver } from "./files.js";
 import { makeOpenTest, loadTests } from "./tests.js";
 
+// TODO The hash changes every time the Rust code does, this is very brittle.
```

Comment:
This will get annoying very quickly. :P  Recommend any other JS build/bundler tool that can do the same sort of asset copying, call `wasm-pack` (until we have an NPM package and/or for local development)?

Reply:
I'll look into this next. There's got to be a good workflow someone's come up with at this point that doesn't bring in an entire framework.

The temporary hack might be a small shell script that calls `wasm-pack` and copies some assets around. This'd be called manually during local dev, and also in the github workflow. Would that be an OK temporary workaround? Or maybe the symlinks are cleaner... I'll see if `wasm-pack` produces consistently named output files.

> That might be worth while in the future, in order to gain access to npm dependencies (without using unpkg), typescript and .less or .scss support, etc.

I actually already hit a real use case for that. This morning, `unpkg` was slow for a bit. And yesterday I didn't have internet on a train, and couldn't develop. Having local caches of all external dependencies and hosting everything ourselves on github pages feels preferable to me.

Reply:
I think a shell script that calls `wasm-pack` and copies files around is a perfect solution. And having a local copy of the JS deps is a good idea too., so let's make a `vendor` dir and store a copy of the JS assets we need in there.

Reply:
I guess you've done that for leaflet and it's been working nicely. Literally should I just `wget` things manually, or does npm/yarn/something help do things like this?

Reply:
OK @BudgieInWA, PTAL. No more trunk. The local dev script literally just calls `wasm-pack` to build the Rust crates to JS/WASM, then runs a Python server. The rest is handled by symlinks. It's nice and simple, works great locally for me. The github action may need a bit more work (I'm unclear if the publish action chases symlinks). I can iterate on that after merging this if needed.

We can vendor external JS deps into this repo as a followup for more performance + less points of failure on other CDNs.